cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java

```diff
@@ -246,7 +247,7 @@ public void getStatus(HttpRequest request, HttpResponder responder,
   private void getScheduleStatus(HttpResponder responder, String appId, String namespaceId, String scheduleName)
     throws NotFoundException, SchedulerException {
     Id.Application applicationId = Id.Application.from(namespaceId, appId);
```

Comment:
convert this to ``ApplicationId`` and use it on line 250, then line 252 will use ``toId``